cayenne/src/main/java/org/apache/cayenne/exp/parser/ExpressionParserTreeConstants.java

```diff
@@ -64,6 +64,7 @@ public interface ExpressionParserTreeConstants
   public int JJTDBPATH = 58;
   public int JJTENUM = 59;
   public int JJTDBIDPATH = 60;
+  public int JJTCUSTOMAGGREGATE = 61;
```

Comment:
It is better to leave generation of this constants to JavaCC and just leave 0 as a new AST node id for now.